src/stories/Forms/InputWrapper/index.tsx

```diff
@@ -132,23 +129,10 @@ export const InputWrapper = React.forwardRef(
       >
         {childrenWithGlobalInputProps}
 
-        {hint && !alert && (
-          <HintWrapper {...HintWrapperProps}>
-            <HintText id={ariaErrorMessageId} className='hint-text'>
-              {hint}
-            </HintText>
-          </HintWrapper>
-        )}
-
-        {alert && (
+        {severity && (
```

Comment:
Maybe also check for `severityMessage` prop, or add default messages for the different severity states?